example/src/App.tsx

```diff
@@ -15,6 +17,9 @@ import {
 } from 'react-native';
 
 import { Room } from './Room';
+import { Typo } from './fonts/Typo';
```

Comment:
I'd put the Typo and  Icon components in a folder named `components`. Then we'd import it like this: `import { Typo } from '@components/Typo'`. Then we'll put other components here. 
Use https://www.npmjs.com/package/babel-plugin-module-resolver for those nice imports.

Reply:
Done.